src/testdir/test_startup.vim

```diff
@@ -109,9 +109,8 @@ func Test_pack_in_rtp_when_plugins_run()
 endfunc
 
 func Test_help_arg()
-  if !has('unix') && has('gui_running')
-    throw 'Skipped: does not work with gvim on MS-Windows'
-  endif
+  CheckNotMSWindows
```

Comment:
This didn't need to be changed, I think.

Reply:
Ah, so this was running on Windows in a terminal?  
I don't have Windows and I'm not familiar with it either. If you think it can be improved. Please create a PR.

Regarding other tests which were added in my change (Test_reverse, Test_background_foreground, Test_font, Test_geometry, Test_iconic) all of them were skipped on Windows because CI was hanging on Windows. Unfortunately, CI logs do not tell which test(s) hang. All I knew from logs, was that `make test_startup` was hanging on Windows). Perhaps some of them can be enabled on Windows. If someone has access to Windows, it would be good to check which tests can be enabled on Windows (possibly with modification to make them pass if needed).

About test hanging: `test_terminal.vim` is also often hanging in CI and we don't know which individual test hangs.

Reply:
> Ah, so this was running on Windows in a terminal?

Yes, this was running before this PR.